.github/workflows/bump-main-loculus-version-to-latest.yaml

```
@@ -37,24 +46,36 @@ jobs:
             ${{ steps.get_current_version.outputs.CURRENT_VERSION }}..${{ steps.get_loculus_commit.outputs.LATEST_SHA }} \
             | sed 's/.*#\([0-9]\+\)[^#]*$/- loculus-project\/loculus#\1/'
           )
+          BREAKING_CHANGES=$( \
+            echo "$CHANGELOG" \
+            | grep -iE '^- (.*!.*|.*breaking.*)' \
+            || true
+          )
           echo "CHANGELOG<<EOF" >> $GITHUB_OUTPUT
           echo "$CHANGELOG" >> $GITHUB_OUTPUT
           echo "EOF" >> $GITHUB_OUTPUT
-      
+          if [ ! -z "$BREAKING_CHANGES" ]; then
+            echo "HAS_BREAKING_CHANGES=true" >> $GITHUB_OUTPUT
+            echo "BREAKING_CHANGES<<EOF" >> $GITHUB_OUTPUT
+            echo "### 🚨 Breaking Changes" >> $GITHUB_OUTPUT
+            echo "$BREAKING_CHANGES" >> $GITHUB_OUTPUT
+            echo "EOF" >> $GITHUB_OUTPUT
+          else
+            echo "HAS_BREAKING_CHANGES=false" >> $GITHUB_OUTPUT
+          fi
       - name: Remove temporary loculus_repo
-        run: |
-          rm -rf loculus_repo
-      
+        run: rm -rf loculus_repo
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v7
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.PATHOPLEXUS_PUSH_PAT }}
           commit-message: "Update Loculus version to https://github.com/loculus-project/loculus/commit/${{ steps.get_loculus_commit.outputs.LATEST_SHA }}"
-          title: "Update Loculus version to ${{ steps.get_loculus_commit.outputs.LATEST_SHA_SHORT }}"
+          title: "${{ steps.changelog.outputs.HAS_BREAKING_CHANGES == 'true' && 'Breaking: ' || '' }}Update Loculus version to ${{ steps.get_loculus_commit.outputs.LATEST_SHA_SHORT }}${{ steps.get_loculus_commit.outputs.IS_CUSTOM_TARGET == 'true' && format(' (non-main: {0})', steps.get_loculus_commit.outputs.TARGET_BRANCH) || '' }}"
```

```suggestion
          title: "${{ steps.changelog.outputs.HAS_BREAKING_CHANGES == 'true' && 'Breaking: ' || '' }}Update Loculus version to ${{ steps.get_loculus_commit.outputs.LATEST_SHA_SHORT }}${{ steps.get_loculus_commit.outputs.IS_CUSTOM_TARGET == 'true' && format(' (non-main: {0})', steps.get_loculus_commit.outputs.TARGET_BRANCH) || '' }}"
```

(This CI is super nice!)
I'm uncertain about whether the PP PR should have `Breaking` in the title - because I think in general we should make the changes needed to unbreak things as part of the same PR. But we could rename the PR after doing that I guess. I don't think we should merge changes to PP with `Breaking` in the title unless stuff is actually broken by them and I don't think we should merge PRs that break stuff :)

---

In particular - I don't think we should have both this _and_ the label below. Then if I fix the breaking change I have to both remember to remove the label _and_ to remove the thing from the title - we should only capture this info in one place.

---

Let's only have the label then? And the PR body contains the breaking anyways!

---

I've removed from title, please rereview :)